pyproject.toml

```diff
@@ -24,7 +24,6 @@ requires-python = '>=3.8'
 dependencies = [
     'abipy',
     'aiida-abinit~=0.4.0',
-    'aiida-bigdft>=0.2.6',
```

Comment:
We had some serious dependency related issues, this is likely a side-effect of the workarounds that were put in place. There were also several interim experimental versions of aiida-bigdft, I'll add back the relevant version

Reply:
re-added it as a requirement, since that was the base version used for the EoS run. There is a more recent rewrite but getting that finalised and pushed to pypi will likely take some time (and also not be relevant to the EoS verification).